### ISSUE TITLE
Property Types

### ISSUE THREAD

**Issue by livingwandsworth** [Markup360/realestatepro#13]
Hi is it possible to add other property types, we will be marketing and selling plots and hotels. 

-Originally asked by jeffscott20 on 24-10-2016, 01:41 PM, migrated from site forum

**Comment by Markup360** [comment]
It's possible but not really straightforward, are you OK to edit some files?

**Comment by livingwandsworth** [comment]
I'm OK with editing........fire away, Im sure all users would like a PLOT option in the search fields.....?

**Comment by Markup360** [comment]
Open this file in any text editor;

realestatepro\inc\custom-fields.php

Then scroll down to line 637 where you will see this block of code;

```
array (
'key' => 'field_5669c4b43e76f',
'label' => 'Property Type',
'name' => 'property_type',
'type' => 'select',
'instructions' => 'Please select the type of property to list.',
'required' => 1,
'choices' => array (
'house' => 'House',
'townhouse' => 'Townhouse',
'flat' => 'Flat',
```


Change any one of the choices to the one you need so you can change 'farm' => 'Farm', to 'plot' => 'Plot', bear in mind that when RealEstatePro is updated this will get overriden so keep a copy of this file with your amendment.

There will be the feature to change the property types from within the admin in the future probably in the next release.

**Comment by livingwandsworth** [comment]
Hi

THanks, I will wait for the new release.

**Comment by Markup360** [comment]
This has been addressed in the latest release v1.0.2.1 and you can now edit the property types.

**Comment by Markup360** [comment]
After you installed the theme you would have received a message at the top to install the plugins that come with RealEstatePro, have you installed them?
